test/E2ETests/compiler/shared/Mocks/StartupSocialTesting.cs

```diff
@@ -103,7 +104,7 @@ public void ConfigureServices(IServiceCollection services)
                 options.ClientId = "[ClientId]";
                 options.ClientSecret = "[ClientSecret]";
                 options.AccessType = "offline";
-                options.Notifications = new GoogleAuthenticationNotifications()
+                options.Notifications = new OAuthAuthenticationNotifications()
```

Comment:
Ah, looks like it is: https://github.com/aspnet/MusicStore/blob/haok/7-1pin/test/E2ETests/compiler/shared/Mocks/StartupSocialTesting.cs#L139